examples/pylint/E0602_undefined_variable.py

```diff
@@ -0,0 +1,9 @@
+"""pylint: Undefined variable %r Used when an undefined variable is accessed.
+
+"""
+
+var1 = 1
+var2 = None
```

Comment:
@hayleyZHL I think I missed this one. Setting a variable to `None` is not the same as an undefined variable. This code should actually run fine; you should delete this line.